pages/login.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Head from 'next/head'
+import LoginForm from '../components/LoginForm'
+import styles from '../styles/Login.module.css'
+import router from 'next/router'
+
+const Login = () => { 
+  const loginRequest = (formData: any) => {
+    fetch("http://localhost:3001/api/login", {

```

Comment:
Check my comment about drying this up in the register page

Reply:
Will work on this. Thank you!